plugins/in_cpu/in_cpu.h

```diff
@@ -123,7 +123,7 @@ static inline double CPU_METRIC_USAGE(unsigned long pre, unsigned long now,
     }
 
     diff = ULL_ABS(now, pre);
-    total = (diff * 100) / ctx->cpu_ticks;
+    total = (diff * 100) / ctx->cpu_ticks / ctx->interval_sec;
```

Comment:
Would you please add some parenthesis to protect cases where a division by zero could happen ?